{"_url": ["core.echo;This+Test"]}
<pre>
Mod_python error: "PythonHandler index"

Traceback (most recent call last):

  File &quot;/usr/lib/python3/dist-packages/mod_python/apache.py&quot;, line 402, in HandlerDispatch
    result = obj(req)

  File &quot;/home/gizmore/PycharmProjects/pygdo/index.py&quot;, line 25, in handler
    method = parser.parse()

  File &quot;/home/gizmore/PycharmProjects/pygdo/gdo/base/Parser.py&quot;, line 27, in parse
    method = self.parse_line(line)

  File &quot;/home/gizmore/PycharmProjects/pygdo/gdo/base/Parser.py&quot;, line 64, in parse_line
    args, unknown_args = parser.parse_known_args(tokens[1:])

  File &quot;/usr/lib/python3.10/argparse.py&quot;, line 1878, in parse_known_args
    namespace, args = self._parse_known_args(args, namespace)

  File &quot;/usr/lib/python3.10/argparse.py&quot;, line 2119, in _parse_known_args
    self.error(_(&#x27;the following arguments are required: %s&#x27;) %

  File &quot;/usr/lib/python3.10/argparse.py&quot;, line 2606, in error
    self.exit(2, _(&#x27;%(prog)s: error: %(message)s\n&#x27;) % args)

  File &quot;/usr/lib/python3.10/argparse.py&quot;, line 2593, in exit
    _sys.exit(status)

SystemExit: 2

</pre>
